contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-rule.y

```diff
@@ -4338,11 +4338,20 @@ tsql_IsolationLevelStr:
 				}
 			| REPEATABLE READ
 				{
-					TSQLInstrumentation(INSTR_UNSUPPORTED_TSQL_ISOLATION_LEVEL_REPEATABLE_READ);
-					ereport(ERROR,
+					if (strcmp(pltsql_repeatable_read_isolation, "pg_isolation") == 0)
+					{
+						TSQLInstrumentation(INSTR_TSQL_ISOLATION_LEVEL_REPEATABLE_READ);
+						$$ = "repeatable read";	
+					}
+					else
+					{
+						TSQLInstrumentation(INSTR_UNSUPPORTED_TSQL_ISOLATION_LEVEL_REPEATABLE_READ);
+						ereport(ERROR,
 							(errcode(ERRCODE_SYNTAX_ERROR),
-							errmsg("REPEATABLE READ isolation level is not supported"),
+							errmsg("Repeatable read is not supported by default, set configuration parameter 'repeatable_read_isolation' to 'pg_isolation' to get PG repeatable read"),
```

Comment:
let us also get approval for error message from PM. It would be sufficient to just point out the config name without specifying which setting to use. Customer can get more details from documentation.